src/main/java/org/reactome/release/dataexport/Main.java

```diff
@@ -1,34 +1,44 @@
 package org.reactome.release.dataexport;
 
-import java.io.FileInputStream;
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+
 import java.io.IOException;
-import java.util.*;
+
+import org.reactome.release.dataexport.configuration.ConfigurationManager;
 
 /**
  * Generates post-release export files for NCBI, UCSC and Europe PMC.
  * @author jweiser
  */
 public class Main {
+	@Parameter(names={"--overwrite-config-file", "-c"})
+	private boolean overwriteConfigFile;
 
 	/**
 	 * Main method to process configuration file and run the executeStep method of the DataExporterStep class
+	 *
 	 * @param args Command line arguments for the post-release data files export (currently the only argument is,
-	 *     optionally, the configuration file
-	 * @throws IOException Thrown if unable to read configuration file, create output directory or write files
-	 * @throws Exception Thrown if execution of executeStep method in DataExporterStep fails
+	 *     optionally, "--overwrite-config-file" or "-c" to indicate the configuration file should be (re)created
+	 * @throws IOException Thrown if unable to create and/or read the configuration file, create output directory
+	 * or write files
 	 */
-	public static void main( String[] args ) throws IOException, Exception {
-		String pathToResources =
-			args.length > 0 ?
-			args[0] :
-			Objects.requireNonNull(
-				Main.class.getClassLoader().getResource("sample_config.properties")
-			).getPath();
+	public static void main( String[] args ) throws IOException {
+		Main main = new Main();
+		JCommander.newBuilder()
```

Comment:
Is JCommander necessary when there is only one possible command-line argument?

Reply:
It's not strictly necessary, but it does make things easier to understand especially if the user accidentally provides invalid arguments.  It also makes things much easier to extend for addtional arguments in the future with little to no cost in code clarity.  @cookersjs and I were discussing adding the ability to run specific exporters rather than all or none as a feature and this would probably use new command-line arguments.

Reply:
Well, if there are plans to add more command-line arguments, then might as well leave JCommander in the project.